tests/DependencyInjection/ConsoleErrorsExtensionExceptionsTest.php

```diff
@@ -68,11 +68,11 @@ public function testExceptionsEnabled(): void
 	 */
 	public function defaultConfigurationValuesDataProvider(): Generator
 	{
-		yield [
+		yield 'listener_priority' => [
```

Comment:
I think we can use the parameter name constant as the case name here. Less writing and more precise.